requirements/py37.yml

```diff
@@ -0,0 +1,28 @@
+name: cf-units-dev
+
+channels:
+  - conda-forge
+
+dependencies:
+  - python 3.7
+
+# setup dependencies
+  - cython
+  - numpy
+  - pytest-runner
```

Comment:
I don't *think* that we need this dependency, right?

Reply:
I'm willing to take your word on this. [Not the first time it has been questioned](https://github.com/conda-forge/cf_units-feedstock/blob/master/recipe/meta.yaml#L23), but I am yet to get involved with `py.test` so I was uncomfortable deciding this myself.

Reply:
67ef259caf5f009c4b394020089d71d900ff07c4